content/blog/2021-11-17-exp-versioning.md

```diff
@@ -0,0 +1,243 @@
+  advantage of all of the features of Git, like diffing and branching.
+- Your entire experiment is codified. Notice that the command above generated a
+  `dvc.yaml` file that tracks the experiment pipeline and all its inputs and
+  outputs.
+
+Of course, you could do all this yourself in Git. Since we have established that
+code version control is insufficient for experiments, DVC adds features specific
+to ML needs.
+
+Large data and models are tracked in your own storage yet are Git-compatible.
+For example, the above command tracks the `models` folder in DVC, making Git
+ignore it yet storing and versioning it for you so that you can back up versions
+anywhere and check them out alongside your experiment code.
+
+Experiments are tracked by DVC so you don't need to create commits or branches
+for each one, yet they are also Git compatible and can be converted into Git
+branches or merged into your existing ones.
+
+```dvc
+$ dvc exp show
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃ Created      ┃    loss ┃    acc ┃ train.epochs ┃
+model.conv_units ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ -            │ 0.25183 │ 0.9137 │ 10           │ 64
+│
+│ dvc-init                │ Oct 23, 2021 │       - │      - │ 10           │ 16
+│
+│ ├── 9a4ff1c [exp-333c9] │ 10:40 AM     │ 0.25183 │ 0.9137 │ 10           │ 64
+│
+│ ├── 138e6ea [exp-55e90] │ 10:28 AM     │ 0.25784 │ 0.9084 │ 10           │ 32
+│
+│ └── 51b0324 [exp-2b728] │ 10:17 AM     │ 0.25829 │ 0.9058 │ 10           │ 16
+│
+└─────────────────────────┴──────────────┴─────────┴────────┴──────────────┴──────────────────┘
+
+$ dvc exp branch exp-333c9 conv-units-64
+Git branch 'conv-units-64' has been created from experiment 'exp-333c9'.
+To switch to the new branch run:
+
+        git checkout conv-units-64
+
+```
+
+Reproducing experiments can be automated since your experiment pipeline is
+codified, and DVC will check for cached inputs and outputs and skip recomputing
+data that's been generated before.
+
+```dvc
+$ dvc exp run
+'data/images.dvc' didn't change, skipping
+Stage 'default' didn't change, skipping
+
+Reproduced experiment(s): exp-44136
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp> <branch>
+```
+
+# What Next?
+
+Experiment versioning is still in its early days. Look out for future
+announcements about:
+
+- Deep learning challenges like live monitoring, checkpointing, and iterative
+  training.
+- Visualizing and comparing experiment results in other tools like VSCode and
+  Studio.
+- Orchestrating experiments on other machines.
+
+What do you want to see for the next generation of experiment tracking? Try out
+experiment versioning, share your thoughts, and help build the future of ML
+developer tools.
```

Comment:
It feels like call to action but without action :) What I'm suppose to do? Comment here, provide a feedback on Discord, tweet with mentioning @DVCOrg?

Or it might be ok to keep it as is 🤷

Reply:
It can be as simple as "What do you want to see for the next generation of experiment tracking? Let us know in the comments!"

Reply:
Good suggestion, @MKhalusova! I used that for now, but we might want to discuss to also link to other channels here (twitter, discord, etc.).